src/js/popup.js

```diff
@@ -687,3 +783,38 @@ $(document).ready(function () {
         }
     });
 });
+
+function appendDomainListEntry (domain) {
+    var $domainListItem;
+    if (domain.default) {
+        $domainListItem = $(
+            `<li class='collection-item container'>
+                <div class="row valign-wrapper remove-margin">
+                    <div id= "custom-domain-label" class="col s10 left-align">
+                        ` + domain.name + ` 
+                    </div>
+                    <div class="col s2">
```

Comment:
This div is empty, but it affects the spacing. I can't get the default domain.names to line up vertically with the non-default domain.names otherwise, even using pushes or pulls. I think I must have a gap in my understanding of the grid system.